modules/helm/options.go

```diff
@@ -15,5 +15,5 @@ type Options struct {
 	EnvVars        map[string]string   // Environment variables to set when running helm
 	Version        string              // Version of chart
 	Logger         *logger.Logger      // Set a non-default logger that should be used. See the logger package for more info.
-	ExtraArgs      []string            // Extra arguments to pass to the helm install/upgrade command
+	ExtraArgs      map[string][]string // Extra arguments to pass to the helm install/upgrade/rollback/delete command
```

Comment:
```suggestion
	ExtraArgs      map[string][]string // Extra arguments to pass to the helm install/upgrade/rollback/delete command. The key signals the command (e.g., install) while the values are the extra arguments to pass through.
```

Reply:
Thanks for the suggestion, I made the change! Resolving this.